code/API_definitions/S2CVPN4(1).yaml

```
@@ -0,0 +1,936 @@
+openapi: "3.0.3"
+info:
+  title: Site to Cloud VPN API 
+  description: |
+    ##### Before starting to use the API, the developer needs to know about the below specified details :
+    * Site to Cloud VPN service endpoint The URL pointing to the RESTful resource of the Site to Cloud VPN API. 
+    * Authentication Security access keys such as OAuth 2.0 client credentials used by client applications to invoke the Site to Cloud VPN API.
+    * Identifier for access point Customer Edge (CE) IPv4 addressand/or IPv6 address of the Customer Edge Router.
+    * Identifier for the endpoint cloud Provider Edge (PE) IPv4 and/or IPv6 address of the Cloud Provider Edge
+    * Identifier for the application server(AS)IPv4 and/or IPv6 address of the application server (cloud virtual machine)
+    * App-Flow (between the application client and application server) The developer wants to have non public network delivery for the precise application data flow for the application users.     
+    * Duration Duration (in hours) for which the Site to Cloud VPN session (between CE and cloud PE) should be created. This parameter is optional. When not specified, a default session duration (e.g. 6 months) is applied. The user may request a termination before its expiration.
+    * Physical Location Physical location of the Customer Edge Router.
+
+
+    NOTE- The private physical link between CE and cloud PE is default deployed for one click calling, otherwise it is required to install the physical link, which is out of the scope of this API.
+
+  version: '0.1'

```

Version is wip until the repository is released, see release process in Release Management and API versioning in chapter 5 of API Design Guidelines in Commonalities:

```suggestion
  version: wip

```